Clinical-Genomics/scout
# Display mitochondrial and rare diseases coverage report in cases with missing track

This PR fixes 2 things:
- Display rare disease coverage report and MT report for RD cases with no saved track
- Fix the bug that causes the case track not to be saved in the database when a case is re-uploaded (fix #2536 and #2507)

**How to test**:
1. Locally, upload the demo case (has "rare" track)
2. From mongo compass, remove the track key from demo case
3. Using master branch, go to demo case page and make sure that MT report is not available in the side panel.
4. From master branch, force reupload the case with the command: 
```scout --demo load case scout/demo/643594.config.yaml -u```
This will cause the case to be reuploaded without a track key, even if the key exists in case the config file
5. MT reports should still be not available from master branch
6. Switch to this branch and make sure that MT is available for the case
7. On this branch, repeat the force upload of the case. This time track : "rare" will be saved in the database for this case
8. MT report should still be present on the sidebar of the case

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
